src/components/Navigation/README.md

```diff
@@ -502,21 +581,24 @@ This example showcases the many elements that can compose a navigation, especial
         {
           url: '/path/to/place',
           label: 'Icon as svg',
-          icon: OnlineStoreMajor,
+          icon: OnlineStoreMinor,
         },
         {
           url: '/path/to/place',
           label: 'Icon as img',
+          shouldResizeIcon: true,
```

Comment:
@alex-page did you say that we shouldn't include `shouldResizeIcon: true`?

Reply:
Yes. I don't see it in main.